uniffi_aries_vcx/core/Cargo.toml

```diff
@@ -19,6 +19,7 @@ path = "uniffi-bindgen.rs"
 uniffi = "0.23.0"
 aries-vcx = { path = "../../aries_vcx" }
 tokio = { version = "1.24.1" }
+libvdrtools = { path = "../../libvdrtools", features=["default"] }
```

Comment:
why is this needed??

Reply:
Was used for accessing the `UnpackMessage` trait for deserialization. Now fixed with an implementation of it inside the uniffi crate.